docroot/themes/custom/su_humsci_theme/su_humsci_theme.info.yml

```diff
@@ -11,6 +11,8 @@ libraries:
 regions:
   page_top: 'Page top'
   header: Header
+  search: Search
```

Comment:
I am curious to the additions of these regions. Why were you not able to accomplish your page layout with the provided themes?

Reply:
So i found out that using the block templates from stanford_basic, the branding and search blocks are wrapped with a decanter-width class. and the search is wrapped with a decanter-end-row class. this really only works correctly if the blocks have the correct weights in relation to each other. So my thought is to add the region which uses the same decanter classes, but the blocks no longer need special classes since its handled in the page template.
Then if we ever decide to add/change a block for that little area, its easier to do so.

The menu region is very similar issue as above. Splitting it out and templated at the page level gives a bit more control over the output.

I can be persuaded

Reply:
Thanks for explaining. Good to have documented. I think this is fine I just wanted to know. I was hoping for less of this but it will be our reality going forward with subthemes and markup classes for display behaviour.